packages/react/src/primitives/PhoneNumberField/CountryCodeSelect.tsx

```diff
@@ -0,0 +1,29 @@
+import { useMemo } from 'react';
```

Comment:
nit: should we add some basic unit tests for this small component? No blocker though

Reply:
Yeah I think that is a great idea. Let me address it in my follow-up PR so we get this component unit-tested before exporting.